### PR TITLE
Work around "Failed to create metal device for video writer"

### PR DESCRIPTION
**Pull request by plu**
On Travis some tests are failing with following error/exception:

    NSInternalInconsistencyException
    Failed to create metal device for video writer.

Skip these tests if the hardware does not support Metal.

### CODE REVIEW

**Comment by plu** [comment]
There's something wrong with the e2e-tests. I will have another look at it tomorrow. Please do not merge yet!

**Comment by plu** [comment]
💤 Travis...

**Comment by plu** [comment]
This is good to go now.

File: fbsimctl/cli-tests/util.py [hunk 1]
```diff
@@ -212,6 +212,13 @@ def launch(self, arguments, timeout=Defaults.TIMEOUT):
             timeout=timeout,
         )
 
+class Metal:
+    def __init__(self):
+        subprocess.call(['xcrun', 'swiftc', 'supports_metal.swift'])
+        self.__supports_metal_exit_code = subprocess.call(['./supports_metal'], stdin=subprocess.DEVNULL, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```
Comment (lawrencelomax) [inline fbsimctl/cli-tests/util.py]: You can use `subprocess.check_call` here instead.

**Pull request closed without merging**